wagtailio/static/sass/components/_headline.scss

```diff
@@ -0,0 +1,109 @@
+.headline {
+    $root: &;
+    position: relative;
+    padding: 80px 0 0;
+    margin-bottom: 50px;
+    overflow: hidden;
+
+    @include media-query(medium) {
+        padding: 130px 0 0;
+        margin-bottom: 100px;
+    }
+
+    @include media-query(large) {
+        padding: 230px 0 0;
+        margin-bottom: 200px;
+    }
+
+    &__heading {
+        position: relative;
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 3;
+        }
+    }
+
+    &__subheading {
+        position: relative;
+        margin-bottom: 20px;
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 4;
+        }
+    }
+
+    &__intro {
+        margin-bottom: 20px;
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+            margin-bottom: 40px;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 3;
+        }
+    }
+
+    &__button-wrap {
+        margin: 0;
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 2;
+        }
+    }
+
+    &__button {
+        display: inline-flex;
+    }
+
+    &__icon {
+        position: absolute;
+        width: 230px;
+        height: 300px;
+        top: 0;
+        right: -40px; // stylelint-disable-line property-disallowed-list
+
+        @include media-query(large) {
+            width: 420px;
+            height: 545px;
+            right: 10%; // stylelint-disable-line property-disallowed-list
```

Comment:
It feels wrong to disable this line, could we use `inset` or remove from the  property-disallowed-list

Reply:
I've enabled the use of it as we use it elsewhere